src/pages/MainPage/MainPage.tsx

```diff
@@ -18,10 +19,24 @@ const MainPage = () => {
     };
   }, {});
 
+  const optionsForSort = {
+    releaseDate: 'Release date',
+    rating: 'Rating',
+  };
+
   return (
     <Layout>
       <div className={styles.filterPanel}>
         <FilterByTags options={optionsForFilter} />
+        <Sort
+          caption='Sort by'
```

Comment:
I can see that you're using a lot of raw user-facing strings which is not extendable and error-prone.
Under "src/core" folder that you create after one of the above comments consider creating a folder "i18y" which stands for "internationalization". This folder in real project usually holds translation object in cases when they're stored in the source code. The folder structure consists of "index.js/ts" file and a number of "*.js/ts" files for different languages (e.g. "en.js" for English or "ge.js" for German). Each of these language files contains an object with the same keys but different values for the target language. The idea behind it is to decouple the selected language for a website and the actual text used in all the components.

E.g. your "Sort" will have an caption text in the JSX as follows:
caption={I18Y['EN'].SORT_BY_CAPTION} for English and caption={I18Y['GE'].SORT_BY_CAPTION} for German. As you might guess, this would allow to hook up language selection logic from your app that will pass either "EN" or "GE" to the I18Y object, but you the same "SORT_BY_CAPTION" key - and end up with different language texts displayed on the live website.

In the end, your task is to create such object (could be only English for now) and go though the whole app and change the hard-coded text to the usage of this I18Y object.